slsim/lensed_population_base.py

```diff
@@ -10,7 +10,9 @@ class LensedPopulationBase(ABC):
 
     def __init__(
         self,
-        sky_area=None,
+        source_sky_area=None,
```

Comment:
what happens to this class if the sky areas =None? If the class can not be initialized with the default option, I would ask at least for partial input without default settings. Just checking